### ISSUE TITLE
Test copy paste to update imports in js/ts

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#235016]
Test for #30066

- [ ] Mac
- [ ] Linux
- [ ] Windows

Complexity: 3

---

# Overview 
This iteration we enabled smart copy/pasting that also updates js/ts imports on paste. This was previously gated behind an experimental setting For this test plan item, test the edits this feature generates along with the UX for how this feature is presented to users

# Testing

### Basic flow
- Try copy and pasting between two TS or JS files that are in the same project

    When you copy code that contains imported symbols, confirm that pasting it shows a paste widget. Selecting `paste with imports` in the paste widget should add the import

- Try copying and pasting different types of code across different files. Confirm that reasonable edits are generated when using `paste with imports`

### Configuration
- Confirm you can use `typescript.updateImportsOnPaste.enabled` / `javascript.updateImportsOnPaste.enabled` to fully disable the feature

    With the feature disabled you should not see the paste widget when pasting

- You can use settings to make `paste with imports` the default:

     - Edit the `editor.pasteAs.preferences` setting
     - Confirm you get intellisense for the various paste types. Specifically you should see `text.updateImports.jsts` listed
     - Add either `text.updateImports.jsts` or just `text.updateImports` to the preferences list
     - Now when pasting, paste with imports should be the default but you can use the paste widget to switch back to normal pasting

# Known limitations
- Smart copy paste currently only works within text editors in the same editor window

**Comment by alexr00** [comment]
Blocked on https://github.com/microsoft/vscode/issues/235096

**Comment by osortega** [comment]
@mjbvz I tested on windows and the only feedback that I have is that I wonder if we should detect if the import already exists and prevent a duplicated import, but not sure if that is out of scope.
Also, looks like the import got inserted in between a comment.

Example:
![Image](https://github.com/user-attachments/assets/26d81539-1330-42c1-bd6a-82145f80a914)

**Comment by mjbvz** [comment]
Thanks @osortega. This bug should track that: https://github.com/microsoft/TypeScript/issues/60419
